cg/meta/upload/nipt/nipt.py

```diff
@@ -24,10 +24,11 @@ def __init__(self, config: CGConfig):
         self.sftp_user: str = config.fluffy.sftp.user
         self.sftp_password: str = config.fluffy.sftp.password
         self.sftp_host: str = config.fluffy.sftp.host
+        self.port = config.fluffy.sftp.port
+        self.remote_path = config.fluffy.sftp.remotepath
```

Comment:
same
```suggestion
        self.sftp_remote_path = config.fluffy.sftp.remotepath
```
And if we are adding this, if its a new config, please use one way to write it. Either 'remote_path' OR 'remotepath'